src/main/java/org/example/bookweb/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,25 @@
 package org.example.bookweb.dto;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
 import lombok.Data;
+import org.example.bookweb.validation.Isbn;
 
 @Data
 public class CreateBookRequestDto {
+    @NotBlank
     private String title;
+    @NotBlank
     private String author;
+    @Isbn
+    @NotBlank
     private String isbn;
+    @NotNull
+    @Positive
     private Double price;
+    @NotBlank
```

Comment:
Let's validate maximum size only. At the moment, your implementation won't allow not to provide a description, but if I remember correctly, it's optional.

Reply:
Can you return the task?